client/app.jsx

```diff
@@ -73,22 +78,102 @@ class App extends React.Component {
     this.setState({allDates: renderedDates});
   }
 
+
   //deals with selecting dates and storing them in state
   selectDate(e, monthYear) {
-    console.log(e, this.state)
-    let selectedDay = e.target.innerHTML;
+    //format selected element data for checking
+    let selectedDay = parseInt(e.target.innerHTML);
```

Comment:
What are you grabbing here with innerHtml?

Reply:
it just grabs the selected day that the user clicks on